tests/antares/services/api_services/test_study_api.py

```diff
@@ -199,3 +199,40 @@ def test_create_binding_constraint_fails(self):
                 match=f"Could not create the binding constraint {constraint_name}: {self.antares_web_description_msg}",
             ):
                 self.study.create_binding_constraint(name=constraint_name)
+
+    def test_read_study_api(self):
+        base_url = "https://antares.com/api/v1"
+        url = f"{base_url}/studies/{self.study_id}"
+        area_url = f"{url}/areas"
+        json_study = {
+            "id": "22c52f44-4c2a-407b-862b-490887f93dd8",
+            "name": "test_read_areas",
+            "version": "880",
+        }
+
+        settings = StudySettings()
+        settings.general_parameters = GeneralParameters(mode="Adequacy")
+
+        config_urls = re.compile(f"https://antares.com/api/v1/studies/{self.study_id}/config/.*")
+
+        with requests_mock.Mocker() as mocker:
+            mocker.get(url, json=json_study)
+            mocker.get(config_urls, json={})
+            mocker.get(area_url, json={})
```

Comment:
You could change this to make it return an area to test even further that the created study has an area